src/main/java/io/github/vcuswimlab/stackintheflow/model/L2H/L2HPredictor.java

```diff
@@ -0,0 +1,182 @@
+package io.github.vcuswimlab.stackintheflow.model.L2H;
```

Comment:
Check how you are doing file io here. There are better ways to get files out of the resources folder other than by using a buffered reader.  Take a look at the TagStatComponent for example, you should really be doing something like `this.getClass().getResourceLoader().getResource()` See https://stackoverflow.com/questions/3861989/preferred-way-of-loading-resources-in-java

Reply:
Thanks for pointing that out, I hadn't considered that could solve around issues with using the jar over a network. I'm now getting resource streams like that. However, I'm accessing said resource streams through an InputStreamReader that is passed to a BufferedReader purely in interest of making minimal changes to the code. Is that okay? Or are there further issues with BufferedReader that I should be weary of?

Reply:
Thanks for pointing that out as well, I hadn't thought about the code paths. I've switched file path logic to use the Path class where possible.